[FEATURE] Ajout du néerlandais sur Pix Site (PIX-11338)

lego-technix opened the pull request:
## :unicorn: Problème

Les sites vitrines ne disposent pas encore de localisation pour le public néerlandophone de Belgique.

## :robot: Proposition

* Ajouter la locale `nl-BE` sur la page d'accueil de pix.org
* Ajouter la locale `nl-BE` dans le composant locale-switcher affiché sur toutes les pages

Cette PR est marquée comme **Blocked** de manière à ne la merger que lorsque l’ordre de déploiement sera donné !

## :rainbow: Remarques

RAS

## :100: Pour tester

1. Tester dans toutes les RA que les comportements, les textes sont bien ceux attendus et qu'il n'y a pas de régressions
2. Tester que les différentes locales sont affichées de la même manière en grande taille d'écran (version bureau) et en petite taille d'écran (version mobile)
3. Une fois que le cookie `locale` a été positionné à une valeur dont il faut se souvenir, se rendre sur https://site-pr631.review.pix.org/ et vérifier qu'on est bien redirigé vers la page de la bonne locale. Cette étape sert à valider que le site est bien capable de gérer la valeur du cookie.

er-lim committed:
feat(site): add nl-be locale in locale choice page

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>

reibecca committed:
refactor: update fr-be label

reibecca committed:
refactor: remove unused keys translation

lego-technix committed:
feat: add nl-be in locale switcher

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>
Co-authored-by: Clément Latzarus <clement.latzarus@pix.fr>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>

reibecca committed:
(fix): remove unused keys translation

Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>

reibecca committed:
refactor(pix-site): replace use of language with locale

reibecca committed:
refactor(pix-pro): replace use of language with locale

er-lim committed:
test: add vue testing library devDependency

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>
Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>

er-lim committed:
test: add integration tests for locale-choice component

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>
Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>

lego-technix committed:
test: add integration tests for locale-switcher component

Co-authored-by: Eric Lim <eric.lim@pix.fr>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>
Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>
Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>

EmmanuelleBonnemay committed:
fix(site): display white page instead of locale choice page if user has already a locale.

lego-technix committed:
test: run pix-site integration tests in CI

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>

lego-technix committed:
test: use "npm test" standard form

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [Pix Site (.fr)](https://site-pr631.review.pix.fr)
  * [Pix Site (.org)](https://site-pr631.review.pix.org)
  * [Pix Pro (.fr)](https://pro-pr631.review.pix.fr)
  * [Pix Pro (.org)](https://pro-pr631.review.pix.org)

Les variables d'environnement seront accessibles sur scalingo https://dashboard.scalingo.com/apps/osc-fr1/pix-site-review-pr631/environment

lego-technix committed:
fix: normalize locale cookie value in BCP 47 format

as stated in https://github.com/1024pix/pix/blob/dev/docs/adr/0040-locales-languages.md

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [Pix Site (.fr)](https://site-pr631.review.pix.fr)
  * [Pix Site (.org)](https://site-pr631.review.pix.org)
  * [Pix Pro (.fr)](https://pro-pr631.review.pix.fr)
  * [Pix Pro (.org)](https://pro-pr631.review.pix.org)

Les variables d'environnement seront accessibles sur scalingo https://dashboard.scalingo.com/apps/osc-fr1/pix-site-review-pr631/environment

igorissen committed:
feat(site): add missing nl-be paths

Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>
Co-authored-by: LEGO Technix <109212476+lego-technix@users.noreply.github.com>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>

igorissen committed:
feat(site): add new npm commands

Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>
Co-authored-by: Rébecca Kaci <rebecca.kaci@gmail.com>

igorissen reviewed (approved):
<img src="https://media1.giphy.com/media/paYMkqI9PFkVSKcSuN/giphy.gif"/>
